.azure-pipelines.yml

```diff
@@ -150,13 +152,13 @@ jobs:
   - task: PublishBuildArtifacts@1
     displayName: 'Publish results'
     inputs:
-      artifactName: 'results'
+      artifactName: 'all-safari-preview'
```

Comment:
@lukebjerring would results-safari-preview be a better name? Not sure what to make this consistent with. All artifacts have results of some kind. But this also isn't *all* tests, just the ones from `./wpt run`. Since every rename here will require changes to the receiver, I'll use whatever name you say and then we won't change it again.

Reply:
Let's stick with results-[product] then.

Reply:
Done!

Reply:
Hmm, don't remember if we said something out of band, but https://github.com/web-platform-tests/wpt.fyi/pull/1107 expects safari-results now.